.github/workflows/go.yml

```
@@ -0,0 +1,27 @@
+name: Go
+on: [push, pull_request]
+#  push:
+#    branches: [ main ]
+#    paths: '**.go'
+#  pull_request:
+#    branches: [ main ]
+#    paths: '**.go'
+jobs:
+  Go:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - run: go list -e -json -compiled=true -test=true -export=true -deps=true -find=false -- ./...
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: latest
+          
+          args: ./...
```

```suggestion
          # args: ./...
```
This seems ___optional___ according to https://github.com/golangci/golangci-lint-action